src/Str.php

```diff
@@ -69,6 +69,10 @@ public static function symmetricSplit($subject, $delimiter, $limit, $default = n
      */
     public static function trimSplit($subject, $delimiter = ',', $limit = null)
     {
+        if ($subject === null) {
```

Comment:
I'd use `empty` here, has the benefit that also an empty string causes an early return.

Reply:
> I'd use `empty` here, has the benefit that also an empty string causes an early return.

Don't forget that `empty('0')` returns `true`.